tbans/controllers/fcm/fcm.py

```diff
@@ -0,0 +1,100 @@
+import json
+import urllib2
+
+from oauth2client.service_account import ServiceAccountCredentials
+
+
+# Instantiate to send FCM message. No initialization required.
+class FCMConnection:
+
+    # Error codes
+    INTERNAL_ERROR = 'internal-error'
+    UNKNOWN_ERROR = 'unknown-error'
+    FCM_ERROR_CODES = {
+        # FCM v1 canonical error codes
+        'NOT_FOUND': 'registration-token-not-registered',
+        'PERMISSION_DENIED': 'mismatched-credential',
+        'RESOURCE_EXHAUSTED': 'message-rate-exceeded',
+        'UNAUTHENTICATED': 'invalid-apns-credentials',
+
+        # FCM v1 new error codes
+        'APNS_AUTH_ERROR': 'invalid-apns-credentials',
+        'INTERNAL': INTERNAL_ERROR,
+        'INVALID_ARGUMENT': 'invalid-argument',
+        'QUOTA_EXCEEDED': 'message-rate-exceeded',
+        'SENDER_ID_MISMATCH': 'mismatched-credential',
+        'UNAVAILABLE': 'server-unavailable',
+        'UNREGISTERED': 'registration-token-not-registered',
+    }
+
+    def __init__(self, project_id='tbatv-prod-hrd'):
+        self._FCM_URL = 'https://fcm.googleapis.com/v1/projects/{project_id}/messages:send'.format(project_id=project_id)
```

Comment:
Another spot where we need to figure out if we can pull that `project_id` dynamically

Reply:
Worst case these should at least be in constants somewhere common.